anta/tests/services.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""
+Test functions related to the EOS various service settings
+"""
+
+# Mypy does not understand AntaTest.Input typing
+# mypy: disable-error-code=attr-defined
+from __future__ import annotations
+
+from typing import List
+
+from pydantic import BaseModel
+
+from anta.custom_types import ErrDisableInterval, ErrorDisableReasons
+from anta.models import AntaCommand, AntaTest
+from anta.tools.utils import get_failed_logs
+
+
+class VerifyErrdisableRecovery(AntaTest):
+    """
+    This class verifies the error-disable recovery reason, status, and interval.
+
+    Expected Results:
+        * Success: The test will pass if the error-disable recovery reason status is enabled and the interval matches the input.
+        * Failure: The test will fail if the error-disable recovery reason is not found, the status is not enabled, or the interval does not match the input.
+    """
+
+    name = "VerifyErrdisableRecovery"
+    description = "Verifies the error-disable recovery reason, status, and interval."
+    categories = ["services"]
+    commands = [AntaCommand(command="show errdisable recovery", ofmt="text")]
```

Comment:
dont you have a json output for this?

EDIT:
4.31
```
ceos1(config)#show errdisable recovery | json
{
    "errors": [
        "This is an unconverted command"
    ]
}
```

Reply:
Yes command not support json output. I was about to add a note in doc string that command does not support json output but missed it. Will add it